tree/dataframe/inc/ROOT/RDF/RLoopManager.hxx

```diff
@@ -196,7 +196,7 @@ class RLoopManager : public RNodeBase {
    // List of branches for which we want to suppress the printed error about
    // missing branch when switching to a new tree. This is modified by readers,
    // so must be declared before them in this class.
-   std::vector<std::string> fSuppressErrorsForMissingBranches{};
+   std::set<std::string> fSuppressErrorsForMissingBranches{};
```

Comment:
Is there a specific reason for using an ordered set?

Reply:
Not in general. But in `GetValidatedColumnNames` it comes handy to have a `std::set` rather than a `std::unordered_set` so that the column names are already sorted for the `std::set_intersection` call. If I change to `std::unordered_set` I would need to define a custom comparator inline. Could be done, but given the size of this set is going to be at worst O(100), I would prefer to leave things as they are.